aquarius/events/decryptor.py

```diff
@@ -8,10 +8,14 @@
 import requests
 from eth_account import Account
 from eth_account.messages import encode_defunct
+from hashlib import sha256
 
 
-def decrypt_ddo(w3, provider_url, contract_address, chain_id, txid):
-    aquarius_account = Account.from_key(os.environ.get("PRIVATE_KEY"))
+def decrypt_ddo(w3, provider_url, contract_address, chain_id, txid, hash):
+    pk = os.environ.get("PRIVATE_KEY", None)
```

Comment:
not sure the event monitor starts without a private key. And either way, the check should not be here. The check should be when starting the event monitor. Here it is... tardive.